### ISSUE TITLE
Add brotli in compression.conf

### ISSUE THREAD

**Issue by Birthright50** [h5bp/server-configs-nginx#231]
It may be worth adding brotli compression, many browsers support this feature.
The option can be optional and commented out by default. (Not all nginx compiled with this module).

**Comment by Llbe** [comment]
I think this would be a good addition, but it should definitely be disabled by default. If I'm not mistaken, there is no official brotli module for Nginx and the official Google module is no longer maintained. This is the most maintained fork of the official Google module: https://github.com/eustas/ngx_brotli

Regarding `brotli_comp_level` I believe 4 or 5 would be a good value. See https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html

**Comment by LeoColomb** [comment]
Thanks for this suggestion @Birthright50 👍 

That said:
* > Not all nginx compiled with this module

  > there is no official brotli module for Nginx

  That the main point. We try to keep our configs compatible with the majority.
* You should prefer using precompressed files instead of on-the-fly compression. (see https://github.com/h5bp/server-configs-apache/issues/105#issuecomment-363739736)

  And when using precompressed files, [only one directive is needed with `ngx_brotli`](https://github.com/google/ngx_brotli#brotli_static):
  ```nginx
  brotli_static on;
  ```

We can certainly add that `brotli_static` line commented out like the `gzip_static` one, but I'm not sure it's worth it given the hard conditions.
In other words, we will accept PR with `brotli_static` addition, but I'm closing this issue since Nginx is not fully compatible with brotli compression right now.

https://github.com/h5bp/server-configs-nginx/blob/28874c33f0e778fc7687f5b3926590ce09e06c98/h5bp/web_performance/compression.conf#L67-L71
